packages/stark-testing/karma.conf.ci.js

```diff
@@ -43,7 +43,8 @@ const rawKarmaConfig = {
 			],
 			resolve: {},
 			transforms: [
-				require("karma-typescript-angular2-transform") // see https://github.com/monounity/karma-typescript-angular2-transform
+				require("karma-typescript-angular2-transform"), // see https://github.com/monounity/karma-typescript-angular2-transform
+				require("karma-typescript-es6-transform")() // add preset in a way that subpackages can find it
```

Comment:
Can you also add the `babel-preset` here the same way you added it in stark-core and stark-ui? `presets: [helpers.root("../stark-testing/node_modules/babel-preset-env")]`

 I know it is taken by default from `node_modules/babel-preset-env` but it is easier to understand what needs to be modified by looking at the options passed to the `"karma-typescript-es6-transform`. And that will be really useful for developers if they check this `karma.conf` file in stark-testing

Reply:
Actually I think that if i put 
```
require("karma-typescript-es6-transform")({
  presets: [helpers.root("./node_modules/babel-preset-env")]
})
```
here I can remove this code from everything that inherits from this file? 🤔

Reply:
I changed it with
```
require("karma-typescript-es6-transform")({
   presets: ["babel-preset-env"] // default setting
})
```

this also works
```
require("karma-typescript-es6-transform")({
   presets: [helpers.root("./node_modules/babel-preset-env")]
})
```